neps/nep-0246.md

```diff
@@ -0,0 +1,550 @@
+---
+NEP: 246 
+Title: Multi Token Standard
+Author: Zane Starr <zane@ships.gold>, @riqi, @marcos.sun
+DiscussionsTo: https://github.com/near/NEPs/discussions/246
+Status: Final
```

Comment:
Looks good @zcstarr - please set this initial state to `Draft` to initiate the NEP process.